corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -117,6 +117,7 @@ def _form_too_large(app, form):
         'templated_intents': domain_has_privilege(domain, privileges.TEMPLATED_INTENTS),
         'custom_intents': domain_has_privilege(domain, privileges.CUSTOM_INTENTS),
         'rich_text': True,
+        'sorted_itemsets': app.build_version >= '2.37',
```

Comment:
Should go https://github.com/dimagi/commcare-hq/blob/45692eb201b822f85a8011656c9b2d79e28f336c/corehq/apps/app_manager/feature_support.py